src/basic/path-util.h

```diff
@@ -17,21 +17,20 @@
 #define PATH_NORMAL_SBIN_BIN(x) x "bin"
 #define PATH_NORMAL_SBIN_BIN_NULSTR(x) x "bin\0"
 
-#if HAVE_SPLIT_BIN
-#  define PATH_SBIN_BIN(x) PATH_SPLIT_SBIN_BIN(x)
-#  define PATH_SBIN_BIN_NULSTR(x) PATH_SPLIT_SBIN_BIN_NULSTR(x)
-#else
-#  define PATH_SBIN_BIN(x) PATH_NORMAL_SBIN_BIN(x)
-#  define PATH_SBIN_BIN_NULSTR(x) PATH_NORMAL_SBIN_BIN_NULSTR(x)
-#endif
+#define DEFAULT_PATH_WITH_SBIN PATH_SPLIT_SBIN_BIN("/usr/local/") ":" PATH_SPLIT_SBIN_BIN("/usr/")
+#define DEFAULT_PATH_WITHOUT_SBIN PATH_NORMAL_SBIN_BIN("/usr/local/") ":" PATH_NORMAL_SBIN_BIN("/usr/")
```

Comment:
BTW, I think "PATH_NORMAL_SBIN_BIN" is a bit hard to grasp. Maybe change that to `PATH_MERGED_BIN` while at it?